Enable go1 to respawn properly

aqubits opened the pull request.

linhdoan8 commented:
@voshch pls check

voshch reviewed (changes requested):
Update .rosinstall with relevant branches

voshch commented:
How to test? Provide launch commands.

aqubits commented:
So, I updated .rosinstall. I added for unitree_ros my repository. 
In order to test it you can do the following:
Firstly, comment the include part of set_ctrl.launch under arena-simulation-setup/robot/go1/launch/control.launch. Then:

$ roslaunch arena_bringup start_arena.launch model:=go1 tm_robots:=scenario tm_obstacles:=scenario tm_modules:=staged simulator:=gazebo

$ roslaunch unitree_controller_go1 set_ctrl.launch robot_namespace:=/go1 target_state:=5

If you have any issue, just let me know.

voshch commented:
Is there a separate arena-simulation-setup branch this works with? The current state gives me
```Timed out waiting for transform from go1/base to odom to become available before running costmap, tf error: canTransform: source_frame go1/base does not exist.```
